crates/forge-runner/src/build_trace_data/test_sierra_program_path.rs

```diff
@@ -0,0 +1,55 @@
+use anyhow::Context;
+use anyhow::Result;
+use cairo_lang_sierra::program::VersionedProgram;
+use camino::{Utf8Path, Utf8PathBuf};
+use fs4::FileExt;
+use std::fs;
+use std::fs::File;
+use std::io::BufWriter;
+
+pub const TESTS_PROGRAMS_DIR: &str = ".snfoundry_test_code";
+
+/// A path to a file with deserialized [`VersionedProgram`] that comes
+/// from compiled test crate. Needed to provide path to source sierra in
+/// [`trace_data::CairoExecutionInfo`].
+#[derive(Clone)]
+pub struct TestSierraProgramPath(Utf8PathBuf);
+
+impl TestSierraProgramPath {
+    pub fn save_sierra_test_program_from_test_crate(
+        versioned_program_from_crate: &VersionedProgram,
```

Comment:
```suggestion
        versioned_program: &VersionedProgram,
```
This is not the part of this function logic